app-template/package-template.json

```diff
@@ -55,12 +55,12 @@
     "angular-mocks": "1.4.10",
     "bezier-easing": "^2.0.3",
     "bhttp": "^1.2.1",
-    "bitauth": "^0.3.2",
+    "bitauth": "git@github.com:gabrielbazan7/bitauth.git#copay",
```

Comment:
For the record, this is temporary until:   https://github.com/bitpay/bitauth/pull/63   is merged.